aten/src/ATen/native/BatchLinearAlgebraKernel.cpp

```diff
@@ -10,6 +10,17 @@
 namespace at { namespace native {
 
 namespace {
+#if AT_BUILD_WITH_LAPACK()
+char to_lapack(TransposeType trans) {
+  switch (trans) {
+    case TransposeType::Transpose: return 't';
+    case TransposeType::NoTranspose: return 'n';
+    case TransposeType::ConjTranspose: return 'c';
```

Comment:
Probably lowercase letters also work, but all the documentation specifies the uppercase, so let's use that.
```suggestion
    case TransposeType::Transpose: return 'T';
    case TransposeType::NoTranspose: return 'N';
    case TransposeType::ConjTranspose: return 'C';
```

Reply:
They both work according to BLAS. I just used lowercase because the `to_blas` used in matmul also used them.
Making them uppercase.